Chapters/01-zig-weird.qmd

```diff
@@ -20,66 +20,51 @@ knitr::opts_chunk$set(
 # Introducing Zig
 
 In this chapter, I want to introduce you to the world of Zig.
-Despite it's rapidly growing over the last years, Zig is, still, a very young language^[New programming languages in general, take years and years to be developed.].
+Zig is a very young language which is actively developed.
 As a consequence, it's world is still very wild and to be explored.
 This book is my attempt to help you on your personal journey for
 understanding and exploring the exciting world of Zig.
 
-I assume you have previous experience with some programming
-language in this book, not necessarily with a low-level one.
-So, if you have experience with Python, or Javascript, for example, is fine.
-But, if you do have experience with low-level languages, such as C, C++, or
-Rust, you will probably learn faster throughout this book.
-
-
+This book won't teach you programming.
+You should have some experience with script languages like Python or Javascript.
+Better if you have experience with a low-level language such as C, C++ or Rust.
 
 ## What is Zig?
 
-Zig is a modern, low-level, and general-purpose programming language. Some programmers interpret
-Zig as the "modern C language". It is a simple language like C, but with some
-modern features.
+Zig is a modern, low-level, and general-purpose programming language. Some programmers think of
+Zig as modern better version of C.
 
-In the author's personal interpretation, Zig is tightly connected with "less is more".
+The author thinks Zig follows the principle of "less is more".
 Instead of trying to become a modern language by adding more and more features,
-many of the core improvements that Zig brings to the
-table are actually about removing annoying and evil behaviours/features from C and C++.
-In other words, Zig tries to be better by simplifying the language, and by having more consistent and robust behaviour.
-As a result, analyzing, writing and debugging applications become much easier and simpler in Zig, than it is in C or C++.
+
+<!-- do you really want to *remove* features from C?? -->
+Zig attempts to simplify and modernize by having more consistency and robustness.
+Its goal is to make writing, reasoning and debugging easier and simpler.
 
 This philosophy becomes clear with the following phrase from the official website of Zig:
 
 > "Focus on debugging your application rather than debugging your programming language knowledge".
 
-This phrase is specially true for C++ programmers. Because C++ is a gigantic language,
-with tons of features, and also, there are lots of different "flavors of C++". These elements
-are what makes C++ so much complex and hard to learn. Zig tries to go in the opposite direction.
-Zig is a very simple language, more closely related to other simple languages such as C and Go.
+<!-- I thought this would be about Zig and not about C++?? -->

```

Comment:
Comparing Zig to C++, or Rust, etc. is important. Making comparisons is actually a common pillar in the learning process. When you learn a new language, you normally learn it by trying to replicate the phrases and words that you already know into this new language.